### ISSUE TITLE
fresh build find no config

### ISSUE THREAD

**Issue by NOOMA-42** [social-tw/social-tw-website#52]
**Describe the bug**
this happens in post-relay branch and branch rebased on it. 
`yarn deploy` generates config.ts in root folder, but to build this branch current commit requires this config.ts which doesn't exist if you didn't run `yarn deploy`

**To Reproduce**
fresh clone test/post-relay branch and build

**Expected behavior**
For information, contracts package.json
```json
"scripts": {
    "build": "yarn buildVerifier && hardhat compile && yarn abi",
    "buildVerifier": "ts-node ./scripts/genVerifier.ts ",
    "abi": "ts-node scripts/abi",
    "hardhat": "hardhat",
    "deploy": "hardhat run scripts/deploy.ts",
```
`yarn deploy` generates config.ts in root folder, but to build this branch current commit requires this config.ts which doesn't exist if you didn't run `yarn deploy`

**Screenshots**
![image](https://github.com/social-tw/social-tw-website/assets/35304302/46721d2f-ee58-4c90-9ea3-67f0b554c3ca)

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**
relates to #25 #47

**Comment by ryanycw** [comment]
Solving this in #25

**Comment by ryanycw** [comment]
This is solved #25
